app/src/org/commcare/dalvik/preferences/CommCarePreferences.java

```diff
@@ -239,9 +239,9 @@ public static int getLoginDuration() {
 
         // default to 24 hours
         try {
-            return Integer.parseInt(properties.getString(LOGIN_DURATION, "28800"));
+            return Integer.parseInt(properties.getString(LOGIN_DURATION, "86400"));
```

Comment:
FWIW:

I prefer setting these constants with something like

24 * 60 * 60

instead of a concrete value, makes it a lot easier for someone reading it to know what the value is